api/stars_api.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import division
+from __future__ import print_function
+
+import logging
+
+import common
+import models
+import notifier
+
+class StarsAPI(common.APIHandler):
+
+  def do_get(self):
+    user = self.get_current_user()
+    if user:
+      feature_ids = notifier.FeatureStar.get_user_stars(user.email())
+    else:
+      feature_ids = []  # Anon users cannot star features.
+
+    data = {
+        'featureIds': feature_ids,
+        }
+    return data
+
+  def do_post(self):
```

Comment:
Please add a comment to explain the logic of this method.  Inferring from the code it's getting the stars for an individual user.  I'd just call it out.

Reply:
Done.